lib/inferno/dsl/runnable.rb

```diff
@@ -464,6 +464,21 @@ def required_suite_options(suite_option_requirements)
           end
       end
 
+      # Replace a child test/group
+      #
+      # @param id_to_replace (Symbol, String) The ID of the child to be replaced
+      # @param new_child (Inferno::TestGroup, Inferno::Test) The new child to replace the existing one
```

Comment:
We use `[` instead of `(` for types everywhere else in the docs, so might as well keep it consistent.